molecule/commands/base.py

```diff
@@ -44,8 +44,12 @@ def __init__(self, command_args, args, molecule=False):
         self.args['<command>'] = self.__class__.__name__.lower()
         self.command_args = command_args
 
-        self.molecule = core.Molecule(self.args)
-        self.main()
+        # allow us to reuse an existing molecule instance
+        if not molecule:
+            self.molecule = core.Molecule(self.args)
+            self.main()
+        else:
+            self.molecule = molecule
```

Comment:
Would be nice to know the use case for this, and if it is even necessary.  I would prefer to leave it out, unless we can duplicate a bug by it not being in.